draft-ietf-masque-connect-ip.md

```diff
@@ -967,6 +969,97 @@ route is restricted to 192.0.2.0/24, rather than 0.0.0.0/0.
 ~~~
 {: #fig-split-tunnel title="VPN Split-Tunnel Capsule Example"}
 
+## Site-to-Site VPN
+
+The following example shows how to connect a branch office network to a
+corporate network such that all machines on those networks can communicate.
+In this example, the IP proxying client is attached to the branch office
+network 192.0.2.0/24, and the IP proxy is attached to the corporate network
+233.252.0.0/24. There are legacy clients on the branch office network that
+only allow maintenance request from machines on their subnet, so the IP
+Proxy is provisioned with an IP address from that subnet.
+
+~~~ aasvg
+
+192.0.2.1 <--+   +--------+             +-------+   +---> 233.252.0.9
+             |   |        +-------------+  IP   |   |
+192.0.2.2 <--+---+ Client | IP Proxying | Proxy +---+---> 233.252.0.8
+             |   |        +-------------+       |   |
+192.0.2.3 <--+   +--------+             +-------+   +---> 233.252.0.7
+
+~~~
+{: #diagram-s2s title="Site-to-site VPN Example"}
+
+In this case, the client does not specify any scope in its request. The IP
+proxy assigns the client an IPv4 address (233.252.0.100) and a split-tunnel
+route to the corporate network (233.252.0.0/24). The client assigns the IP
+proxy an IPv4 address (192.0.2.200) and a split-tunnel route to the branch
+office network (192.0.2.0/24). This allows hosts on both networks to
+communicate with each other, and allows the IP proxy to perform maintenance
+on legacy hosts in the branch office.
+
+~~~
+[[ From Client ]]             [[ From IP Proxy ]]
+
+SETTINGS
+  H3_DATAGRAM = 1
+
+                              SETTINGS
+                                ENABLE_CONNECT_PROTOCOL = 1
+                                H3_DATAGRAM = 1
+
+STREAM(44): HEADERS
+:method = CONNECT
+:protocol = connect-ip
+:scheme = https
+:path = /corp
+:authority = proxy.example.com
+capsule-protocol = ?1
```

Comment:
Shouldn't there be a 401 with WWW-Authenticate and then a repeat of the request with Authorization header here?

Reply:
We explicitly declare Authentication out of scope so I don't think we need it in the examples

Reply:
Ok.